### PR TITLE
Modify the xrandr parser to extract display model info from EDID #364

### PR DESCRIPTION
**Pull request by tzeikob**

### CODE REVIEW

**Comment by kellyjonbrazil** [comment]
Thanks for submitting! I’ll take a closer look in the next few days.

**Comment by kellyjonbrazil** [comment]
Looks like tests are failing on older versions of python (3.7, 3.6) due to some modern type annotations used. Unfortunately we need to be backward compatible to python 3.6 until Centos7 is no longer supported. Could you modify/remove the offending type annotations?

File: jc/parsers/xrandr.py [hunk 1]
```diff
@@ -294,15 +375,67 @@ def _parse_device(next_lines: List[str], quiet: bool = False) -> Optional[Device
...
+#      010101010101381d56d4500016303020
+#      250058c2100000190000000f00000000
+#      000000000025d9066a00000000fe0053
+#      414d53554e470a204ca34154000000fe
+#      004c544e313536415432343430310018
+_edid_head_pattern = r"\s*EDID:\s*"
+_edid_line_pattern = r"\s*(?P<edid_line>[0-9a-fA-F]{32})\s*"
+
+
+def _parse_model(next_lines: List[str], quiet: bool = False) -> Optional[Model]:
```
Comment (kellyjonbrazil) [inline jc/parsers/xrandr.py:409]: It looks like the type annotation here (specifically referring to `Model`) is breaking python versions 3.6 and 3.7. Not sure why, but maybe we just modify/remove the type annotation here and see if that will allow the tests to pass? Type annotations seem to be a bit broken in this parser, overall.

Comment (tzeikob) [reply jc/parsers/xrandr.py:409]: Sure, we can do that.

The only files the xrandr parser needs, in order to decode the EDID data, is the Edid and EdidHelper classes. I think we can remove any other file entirely and keep those classes having them modified a bit by removing any @staticmethod, @classmethod annotations.

Let me know if there is a way to run the tests which are failing for centos to save some from back and forth time. I suppose the `runtests.sh` script does not include those tests, right? I ran this script and all test are passing.

Comment (kellyjonbrazil) [reply jc/parsers/xrandr.py:409]: All the tests run with `./runtests.sh`, but github actions just runs the same tests several times with different Python versions and operating systems. The way to do it on your local machine is by using something like [pyenv](https://github.com/pyenv/pyenv) so you can change to an older Python version and run `./runtests.sh` again.

**Comment by tzeikob** [comment]
@kellyjonbrazil I think I've found the root of the cause.

I was actually missed entirely the part where typed dictionaries must be declared as generic ones in case the TypedDict is not supported, as it turns out for older versions of python.

**Comment by kellyjonbrazil** [comment]
Looks great - I'll go ahead and merge. Thank you for the contribution!

**Pull request merged**